plugins/jitsi-videobridge/jitsi-videobridge

```diff
@@ -0,0 +1,132 @@
+#!/bin/sh
+
+set -e
+
+: << =cut
+
+=head1 NAME
+
+jitsi-videobridge - Monitor sessions and conferences 
+on a jitsi-videobridge
+
+=head1 APPLICABLE SYSTEMS
+
+Jitsi-Videobridge instances
+
+=head1 CONFIGURATION
+
+Requires enabled colibri statistics on jitsi-videobridge and an 
+installed jq, a command-line json processor.
+
+You may specify the URL where to get the statistics
+
+[jitsi-videobridge]
+env.url http://127.0.0.1:8080/colibri/stats
+
+... and you may disable the audiochannel when you don't use an
+audio gateway.
+
+=head1 AUTHOR
+
+Copyright (C) 2020 Sebastian L. (https://momou.ch)
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=manual
+ #%# capabilities=autoconf
+
+=cut
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+COLIBRI_URL=${url:-"http://127.0.0.1:8080/colibri/stats"}
+
+case $1 in
+
+    autoconf)
+        if [ -x /usr/bin/curl ]; then
+            if [ -x /usr/bin/jq ]; then
+                echo "yes"
+                exit 0
+            else
+                echo "no (jq not found)"
+                exit 0
+            fi
```

Comment:
For a perfectly self-configuring plugin you should add a test for an empty result of `curl "$COLIBRI_URL"` here. In this case you can also use the family `auto`.

btw: the `/stats` part of the URL path is probably based on the API of the jitsi service. Maybe you want to let the configurable part of the URL only be `http://localhost:8080/colibri` and add the `/stats` path in the plugin. This would allow the plugin to re-adjust the requested path, in case of changes in the jitsi API.

But this is a matter of taste, I guess.